pages/about.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/grid.css">
+    <title>More About Me!</title>
 </head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+<div class="bg"></div>
+<div class="bg bg2"></div>
+<div class="bg bg3"></div>
+<!-- <div class="content"> -->
+  <!-- <h1>Sliding Diagonals Background Effect</h1> -->
+<!-- </div> -->
+
+<div class="wrapper">
+    <div class="grid-container">
+        <header class="grid-header">
+        </header>
+
+        <nav class="grid-top-nav">
+            <ul class="grid-nav-ul">
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="about"><a href="about.html">About</a></li>
+                <li id="portfolio"><a href="portfolio.html">Portfolio</a></li>
+            </ul>
+        </nav>
+
+        <section class="grid-main">
+
+            <div class="grid-about-content">
+                <p class="about-page-bio">My name is Marjan Abdollahi, my pronouns are she/her. I studied accounting, before changing my career in software development, I worked in the Bank and also for a few years I was a stock trader. 
```

Comment:
There is no `</p>` tag on this paragraph.  This won't register as an error, because 'p' tags can not contain block level elements, so they will automatically close if followed by a block level element.  Since `p` itself is a block level element, this block will automatically close when the following `p` block begins.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p